docs/zh/latest/admin-api.md

```diff
@@ -39,7 +39,7 @@ title: Admin API
 
 Admin API 是为 Apache APISIX 服务的一组 API，我们可以将参数传递给 Admin API 以控制 APISIX 节点。更好地了解其工作原理，请参阅 [architecture-design](./architecture-design/apisix.md) 中的文档。
 
-启动 Apache APISIX 时，默认情况下 Admin API 将监听 `9080` 端口（HTTPS 的 `9443` 端口）。您可以通过修改 [conf/config.yaml](../../../conf/config.yaml) 文件来改变默认监听的端口。
+启动 Apache APISIX 时，默认情况下 Admin API 将监听 `9080` 端口（HTTPS 的 `9443` 端口）。您可以通过修改 [conf/config.yaml](https://github.com/apache/apisix/blob/master/conf/config.yaml) 文件来改变默认监听的端口。
```

Comment:
Why not use a relative link? We cannot redirect to this doc in the current branch if we use this link.

Reply:
Okay

Reply:
Since the same change was made to Eng docs It isn't  fixed .I don't know why still  Page Not Found 😕😕
![ki](https://user-images.githubusercontent.com/53715187/125627096-2c4dfba1-2f74-4c55-bbb6-0713be02d5b4.jpg)

Reply:
Maybe @juzhiyuan can help?

Reply:
@KishaniKandasamy see this link https://docusaurus.io/docs/docs-markdown-features , and then try searching `docusarus relative links`, if still not work, let me know.

Reply:
Ok👍